README.md

```diff
@@ -21,127 +47,131 @@ c2patool is a command line tool for working with C2PA [manifests](https://c2pa.o
+| `--detailed` | `-d` | N/A | Display detailed C2PA-formatted manifest data.  See [Displaying a detailed manifest report](#displaying-a-detailed-manifest-report). |
+| `--force` | `-f` | N/A | Force overwriting output file. See [Forced overwrite](#forced-overwrite). |
+| `--help` | `-h` | N/A | Display CLI help information. |
+| `--output` | `-o` | `<output_file>` | Specifies path to output file. See [Adding a manifest to an asset file](#adding-a-manifest-to-an-asset-file). |
+| `--manifest` | `-m` | `<manifest_file>` | Specifies a manifest file to add to an asset file. See [Adding a manifest to an asset file](#adding-a-manifest-to-an-asset-file).
+| `--parent` | `-p` | `<parent_file>` |  Specifies path to parent file. See [Specifying a parent file](#specifying-a-parent-file). |
+| `--remote` | `-r` | `<manifest_url>` | Specify remote manifest available over HTTP. See [Generating a remote manifest](#generating-a-remote-manifest)|
+| `--sidecar` | `-s` | N/A | Put manifest in external "sidecar" file with `.c2pa` extension. See [Generating an external manifest](#generating-an-external-manifest). |
+| `--version` | `-V` | N/A | Display version information. |
 
 ### Displaying manifest data
 
-Invoking the tool with a path to an asset will print a report describing the manifests contained in the file in JSON format to stdout.
+To display the manifest associated with an asset file, provide the relative path to the file as the argument; for example:
 
 ```shell
 c2patool sample/C.jpg
 ```
 
-### Detailed manifest report
+The tool displays the manifest JSON to standard output (stdout).
+
+To display a detailed report describing the internal C2PA format of manifests contained in the asset, use the `-d` option; for example:
 
-The `-d` or `--detailed` option will print a detailed report describing the internal C2PA format of manifests contained in the file in JSON format to stdout.
 ```shell
-c2patool sample/C.jpg -d
+c2patool -d sample/image.jpg
 ```
 
-### Adding a manifest to a file
-
-To add C2PA data to a file, use the `-m` or `--manifest` option and provide a manifest definition JSON file as the argument. The tool generates a new manifest using the values given in the file.
-
-The file path must reference the asset to be signed. 
+The tool displays the detailed report to standard output (stdout).
 
-The output file is specified on the command line via the `-o` or `--output` flag.  If the output is the same as the source, it will be overwritten. Use this with caution. If no output is given you can preview the generated manifest but nothing is written.
+### Adding a manifest to an asset file
 
-The generated manifest store will also be reported in JSON format to stdout.
+To add C2PA manifest data to a file, use the `--manifest` (or `-m`) option with a manifest JSON file as the option argument and the path to the asset file to be signed.  Specify the output file as the argument to the `-o` or `--output` option.   For example:
```

Comment:
Are we doing two spaces after periods? Or one? Or three?

My vote: 1.

Reply:
I agree.   

I don't think it's that significant, and I confess I don't usually worry about it in markdown, since HTML ignores extra whitespace and renders it as a single space.  But we might as well be consistent when it's at hand.